dotlottie-rs/src/lottie_renderer/mod.rs

```diff
@@ -31,10 +31,14 @@ pub struct LottieRenderer {
 
 impl LottieRenderer {
     pub fn new() -> Self {
-        let thorvg_canvas = Canvas::new(TvgEngine::TvgEngineSw, 0);
+        let mut thorvg_canvas = Canvas::new(TvgEngine::TvgEngineSw, 0);
```

Comment:
Maybe you can remove these dummy instances, you can set an LottieRenderer initialization flag (bool) to quickly bypass the calls to the other requested methods. The initialization toggle should be assigned within the `load_data()` method.

Reply:
Alternatively, you can reuse the background and animation with calling clear(false) in the load method.

Reply:
Great suggestion @hermet! 

I was planning to reuse the same instances to avoid new mem allocation on load, i added this temp workaround for testing.